src/org/labkey/test/components/ui/grids/TabbedGridPanel.java

```diff
@@ -56,6 +58,25 @@ public List<String> getTabsWithoutCounts()
                 .collect(Collectors.toList());
     }
 
+    public Map<String, Integer> getTabsWithCounts()
+    {
+        Map<String, Integer> counts = new HashMap<>();
```

Comment:
Use a `LinkedHashMap` to retain tab order.
```suggestion
        Map<String, Integer> counts = new LinkedHashMap<>();
```

Reply:
Updated.